### DISCUSSION TITLE
github win10 ssh

### DISCUSSION THREAD

**Discussion by horvathzoltan** [community/community#35423] [New to GitHub]
hi github guys! it would be nice to actualize github docs on "how to generate and use ssh keys on win10" 'couse it is tricky a bit. you should to start OpenSSH Authentication Agent by hand, 'couse it is disabled by default, it would be nice to write a few word about it in the relevant sections of the docs. thx, bye!

**Comment by queenofcorgis** [comment]
Hi there! Thanks for this feedback - any GitHub user can submit to our docs 🚀 If you'd like to scroll to the bottom of the doc and use the Make a Contribution button!
